### PR TITLE
Added ability to choose the currency pair

### PR DESCRIPTION
**Pull request by pirhoo**
Hi,

I added an extra option to the `.env` to be able to chose the traded currency pair. So now you can deal with every currencies present on Kraken.

Hope it helps,

Pierre

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -37,25 +49,25 @@ const investmentAmount = process.env.INVESTMENT_AMOUNT;
         });
         // buy disposed amount for today
         const tradeResponse = await client.api('AddOrder', {
-            pair: 'XBTCZEUR',
-            volume: btcToBuy,
+            pair,
```
Comment (0x13a) [inline index.js]: what about the pair here?

Comment (0x13a) [inline index.js]: `XXBTZEUR` vs `XBTCZEUR ` do you know if this affect the order base currency?

Comment (0x13a) [reply index.js]: there is not even a sandbox for kraken, i will have to dev test with real money. I've taken the `XBTCZEUR ` from the _"official"_ kraken php sdk

Comment (0x13a) [reply index.js]: also this should be `pair: pair`

Comment (pirhoo) [reply index.js]: With ECMAScript 2015 you can use this shorter notation:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions

I tried to do a transaction with `XLTCZEUR` and it worked fine :)

Comment (0x13a) [reply index.js]: cool thanks for the link.

Comment (pirhoo) [reply index.js]: Thanks!

**Pull request merged**